src/Network/Network/help/New-AzNetworkManagerSecurityAdminConfiguration.md

```diff
@@ -213,6 +214,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: True
 ```
 
+### -ProgressAction
```

Comment:
Add new property here and add the networkGroup option in the admin rule file. Please also add examples for each.